instrumentation/logback-classic-1.2/src/main/java/com/nr/agent/instrumentation/logbackclassic12/Logger_Instrumentation.java

```diff
@@ -0,0 +1,19 @@
+package com.nr.agent.instrumentation.logbackclassic12;
+
+import ch.qos.logback.classic.Level;
+import com.newrelic.api.agent.NewRelic;
+import com.newrelic.api.agent.weaver.MatchType;
+import com.newrelic.api.agent.weaver.Weave;
+import com.newrelic.api.agent.weaver.Weaver;
+import org.slf4j.Marker;
+
+@Weave(originalName = "ch.qos.logback.classic.Logger", type = MatchType.Interface)
+public abstract class Logger_Instrumentation {
+
+    private void buildLoggingEventAndAppend(final String localFQCN, final Marker marker, final Level level, final String msg, final Object[] params,
+            final Throwable t) {
+        Weaver.callOriginal();
+        NewRelic.incrementCounter("Logging/lines");
+        NewRelic.incrementCounter(String.format("Logging/lines/%s", level));
```

Comment:
Typically we want to do New Relic specific stuff first and then return execution to the original code, which means that the `Weaver.callOriginal()` should come last. I assume that the value of `level` isn't dependent on the original code executing first and that it's merely the value that was passed in to this method as an argument? If that's true, then let's move the NewRelic API calls ahead of `Weaver.callOriginal()`.